Core/StitchCoreSDK/Sources/StitchCoreSDK/Internal/Net/StitchRequestClient.swift

```diff
@@ -7,9 +7,9 @@ import MongoSwift
 private func inspectResponse(response: Response) throws -> Response {
     guard response.statusCode >= 200,
         response.statusCode < 300 else {
-        throw StitchErrorCodable.handleError(forResponse: response)
+            throw StitchErrorCodable.handleError(forResponse: response)
```

Comment:
[nit] I think this was previously at the correct indentation level. the guard statement could probably fit on one line making that clearer.

Reply:
fixed